services/productService.js

```diff
@@ -0,0 +1,14 @@
+const { productDao } = require("../models");
+
+const getAllMainCategory = async (subCategoryId) => {
+  return await productDao.getAllSubCategory(subCategoryId);
```

Comment:
```suggestion
  return await productDao.getAllMainCategory(subCategoryId);
```
모든 서비스 메소드가 전부 SubCategory를 호출하고 있는 것 같네요! 
PR을 생성하신 뒤에 `Files changed`를 한 번 더 체크해주세요!